### PR TITLE
Added a README file

### LINKED ISSUE #2148: Insertion Sort
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Dart
- [x] Go
- [x] Java
- [x] JS
- [x] Kotlin
- [ ] Php - @saloniankita 
- [x] Python
- [ ] Readme - @debdutgoswami
- [x] Ruby
- [x] Rust - @m-code12
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by Raksha25**
Fixes #2148 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added a readme file.

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Raksha25** [comment]
@jainaman224  kindly review my PR.

File: Insertion_Sort/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+[coffee code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.coffee>
+  [cpp code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.cpp>
+   [c# code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.cs>
+   [go code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.go>
+   [java code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.java>
+   [javaScript code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.js>
+   [kotlin code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.ky>
+   [python code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.py>
+   [ruby code]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.rb>
+   [dart code ]: <https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Insertion_Sort/Insertion_Sort.dart>
```
Comment (raghav-dalmia) [inline Insertion_Sort/README.md]: some of the links are not working

File: Insertion_Sort/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+5, 11, 12, 13, 6
+-	i = 4. 6 will move to position after 5, and elements from 11 to 13 will move one position ahead of their current position.
+5, 6, 11, 12, 13
+
+# Time Complexity
+O(n^2)
+
+# Implementation
+
+ - # [c code]
```
Comment (raghav-dalmia) [inline Insertion_Sort/README.md]: use simple text instead of `#`

File: Insertion_Sort/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  - It compares the current element with the largest value in the sorted array. If the current element is greater, then it leaves the element in its place and moves on to the next element else it finds its correct position in the sorted array and moves it to that position. This is done by shifting all the elements, which are larger than the current element, in the sorted array to one position ahead.
+ 
+#   Algorithm
+### Input
+Array and array size.
+### Output
+Sorted Array.
+### Steps
+insertionSort(arr, n)
```
Comment (raghav-dalmia) [inline Insertion_Sort/README.md]: indentation issue in the output file.

File: Insertion_Sort/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+-	i = 2. 13 will remain at its position as all elements in A[0..i-1] are smaller than 13
+11, 12, 13, 5, 6
+-	i = 3. 5 will move to the beginning and all other elements from 11 to 13 will move one position ahead of their current position.
+5, 11, 12, 13, 6
+-	i = 4. 6 will move to position after 5, and elements from 11 to 13 will move one position ahead of their current position.
+5, 6, 11, 12, 13
+
+# Time Complexity
+O(n^2)
+
```
Comment (raghav-dalmia) [inline Insertion_Sort/README.md]: include space complexity section

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this issue

**Pull request closed without merging**